apps/builder/app/builder/shared/css-editor/copy-paste-menu.tsx

```diff
@@ -13,21 +13,20 @@ import {
   toValue,
   type StyleMap,
 } from "@webstudio-is/css-engine";
-import { useStore } from "@nanostores/react";
-import { $advancedStylesLonghands } from "./stores";
 
 export const copyAttribute = "data-declaration";
 
 export const CopyPasteMenu = ({
   children,
   properties,
+  styleMap,
   onPaste,
 }: {
   children: ReactNode;
   properties: Array<string>;
+  styleMap: StyleMap;
```

Comment:
maybe use custom type Map<CssProperty, StyleValue>?

Reply:
kinda wanted to keep the central place for StyleMap, but maybe we can just add CssMap there and deprecate StyleMap so that its clear where this is going?

Reply:
or what's a better name? CssStyleMap?

Reply:
I mean we don't need it in css-engine. This is css-editor specific api

Reply:
done